Configurations/10-main.conf

```diff
@@ -653,6 +658,7 @@ my %targets = (
         shared_cflag     => "-fPIC",
         shared_ldflag    => "-Wl,-znodelete",
         shared_extension => ".so.\$(SHLIB_VERSION_NUMBER)",
+        rpath_flag       => "-Wl,-rpath,",
```

Comment:
For consistency (since there's a no-self `shared_target` here), so should this.
(although I don't think it would be a bad thing to move the linux-shared data from `shared-info.pl` here, but that's not really in scope for this PR)